app/views/govuk_publishing_components/components/layout_header/_header_logo.html.erb

```diff
@@ -0,0 +1,25 @@
+
+<div class="govuk-header__logo gem-c-header__logo">
+  <a href="/" class="govuk-header__link govuk-header__link--homepage">
+    <span class="govuk-header__logotype">
+      <svg role="presentation" focusable="false" class="govuk-header__logotype-crown" xmlns="http://www.w3.org/2000/svg" viewbox="0 0 132 97" height="32" width="36">
```

Comment:
role=presentation is be replaced with aria-hidden=true
see https://github.com/alphagov/govuk-frontend/pull/1724